packages/eslint-plugin-pf-codemods/lib/rules/v5/datepicker-warn-helperText.js

```diff
@@ -0,0 +1,30 @@
+const { getPackageImports } = require("../../helpers");
+
+  // https://github.com/patternfly/patternfly-react/pull/8833
+  module.exports = {
+    meta: {},
+    create: function (context) {
+      const imports = getPackageImports(
+        context,
+        "@patternfly/react-core"
+      ).filter((specifier) => specifier.imported.name === "DatePicker");
+
+      return imports.length === 0
+        ? {}
+        : {
+            JSXOpeningElement(node) {
+              if (
+                imports
+                  .map((imp) => imp.local.name)
+                  .includes(node.name.name)
+                  && node.attributes.find(a => a.name.name === 'helperText')
+              ) {
+                context.report({
+                  node,
+                  message: 'The helperText property now accepts the <HelperText> component.',
```

Comment:
Based on the latest updates to the associated react PR, this message will need to be updated since now whatever is passed to helperText is automatically wrapped in HelperText components.

Reply:
The Datepicker PR now does not automatically wrap the prop. I will update the message to say "expects" instead.